ckanext/datagovtheme/templates/templates_new/organization/read_base.html

```diff
@@ -32,7 +32,7 @@
 {% endblock %}
 
 {% block secondary_content %}
-  {% snippet 'snippets/organization.html', organization=c.group_dict, show_nums=true, bureau_codes=request.args.getall('bureauCode') %}
+  {% snippet 'snippets/organization.html', organization=c.group_dict, show_nums=true, bureau_codes=request.args.get('bureauCode') %}
```

Comment:
This may change to `bureau_code` depending on how this interacts with dcat_usmetadata and usmetadata..